packages/core/src/application.ts

```diff
@@ -207,3 +211,14 @@ export interface ApplicationConfig {
 
 // tslint:disable-next-line:no-any
 export type ControllerClass = Constructor<any>;
+
+/**
+ * Type description for `package.json`
+ */
+export interface PackageJson {
```

Comment:
JSON is a serialization format. Can we replace it with something else in the interface name please? How about `PackageManifest`, `PackageMetadata` or even `ApplicationManifest`/`ApplicationMetadata`?

I think I like `ApplicationMetadata` most.